src/d3a/models/strategy/area_agents/two_sided_pay_as_clear_engine.py

```diff
@@ -105,8 +104,8 @@ def _match_offers_bids(self):
             return
         clearing_rate, clearing_energy = clearing
         if clearing_energy > 0:
-            self.owner.log.info(f"Market Clearing Rate: {clearing_rate} "
-                                f"||| Clearing Energy: {clearing_energy} ")
+            self.owner.log.warning(f"Market Clearing Rate: {clearing_rate} "
```

Comment:
I would suggest to revert it, since it does not have anything to do with this story. There was also quite a long discussion about that in a past review, and I am under the impression we decided to keep it to info level.

Reply:
Sorry, left it there mistakenly. Had to visualize the clearing_rate in logs.